<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timeSeriesDialog</class>
 <widget class="QDialog" name="timeSeriesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Series Plots</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>344</y>
     <width>545</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>294</y>
     <width>531</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <widget class="QPushButton" name="pushButtonHelp">
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonSavePlot">
      <property name="text">
       <string>Save Plot</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonClose">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonPlot">
      <property name="text">
       <string>  Plot  </string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>12</y>
     <width>531</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>8</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Time Series Feature Type</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>36</y>
      <width>471</width>
      <height>181</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>  Element  </string>
     </attribute>
     <widget class="QComboBox" name="comboBoxEleFeature">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>59</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Element ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Average All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Average Element IDs</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>65</y>
        <width>141</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Features by .....</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxEleVariable">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>14</y>
        <width>221</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Interception Storage</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Saturated State</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Unsaturated State</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Surface State</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Evaporation from Canopy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tranpiration</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Evaporation from Surf/Soil</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Precipitation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Net Precipitation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Infiltration</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Groundwater Recharge</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>20</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Plot Variable:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>110</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Default is the same as Model Output Interval Selected. However user can specify multiple of the default time step.</string>
      </property>
      <property name="text">
       <string>Time Step</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEleID">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>62</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string comment="Enter Feature ID">Enter Feature ID</string>
      </property>
      <property name="text">
       <string>Enter ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEleTime">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string comment="Enter Feature ID">Enter Time-Average Interval</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>291</x>
        <y>107</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>Minutes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hours</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Days</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>   Stream    </string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>68</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Features by .....</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxRivFeature">
      <property name="geometry">
       <rect>
        <x>188</x>
        <y>63</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Reach ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Average All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Average Reach IDs</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>23</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Plot Variable:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>112</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Default is the same as Model Output Interval Selected. However user can specify multiple of the default time step.</string>
      </property>
      <property name="text">
       <string>Time Step</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditRivID">
      <property name="geometry">
       <rect>
        <x>359</x>
        <y>65</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string comment="Enter Feature ID">Enter Feature ID</string>
      </property>
      <property name="text">
       <string>Enter Riv ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditRivTime">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>111</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string comment="Enter Feature ID">Enter Time-Average Interval</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxRivVariable">
      <property name="geometry">
       <rect>
        <x>189</x>
        <y>15</y>
        <width>221</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Head</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Inflow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Outflow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Base Flow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Surface Flow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Base Flow Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Base Flow Right</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Surface Flow Left</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Surface Flow Right</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>291</x>
        <y>107</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>Minutes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hours</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Days</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEditFileName">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>234</y>
      <width>231</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>236</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The Simulation Folder contains the model simulation results from which plots are generated. Note, there may be multiple simulation folders (eg. Simul00, Simul01, Simul02 etc).</string>
    </property>
    <property name="text">
     <string>Simulation Folder</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>416</x>
      <y>225</y>
      <width>98</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="pushButtonBrowse">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>timeSeriesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
